experiment/config-forker/main.go

```diff
@@ -252,6 +252,38 @@ func fixEnvVars(vars []v1.EnvVar, version string) []v1.EnvVar {
 	return newVars
 }
 
+func fixTestgridAnnotations(annotations map[string]string, version string, isPresubmit bool) map[string]string {
+	r := strings.NewReplacer(
+		"master-blocking", version+"-blocking",
+		"master-informing", version+"-informing",
+	)
+	a := map[string]string{}
+	didDashboards := false
+annotations:
+	for k, v := range annotations {
+		switch k {
+		case "testgrid-dashboards":
```

Comment:
can we pull the testgrid annotation names into constants to ease refactoring

Reply:
done.